buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -222,100 +173,60 @@ until [ $daemon_status == "Synced" ]; do
 mina-dev accounts unlock --public-key $BLOCK_PRODUCER_PK
 mina-dev accounts unlock --public-key $SNARK_PRODUCER_PK
 
-# Start sending payments
-send_payments() {
+# Start sending value transfer transactions
+send_value_transfer_txns() {
   mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -nonce 0 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
   while true; do
     sleep $TRANSACTION_FREQUENCY
     mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
   done
 }
-send_payments &
-
-# Fee payer cache creation
-echo "==================== PREPARE FEE PAYER CACHE ======================"
-zkapp_fee_payer_pk=$(cat ${MINA_KEYS_PATH}/zkapp-${zkapp}-fee-payer.key.pub)
-zkapp_fee_payer_privkey=$(mina-dev advanced dump-keypair --privkey-path "${MINA_KEYS_PATH}/zkapp-${zkapp}-fee-payer.key" | sed -ne "s/Private key: //p")
-
-mkdir -p /root/.cache/zkapp-cli/keys
-echo -e "{\n    \"privateKey\": \"${zkapp_fee_payer_privkey}\",\n    \"publicKey\": \"${zkapp_fee_payer_pk}\"\n}" >/root/.cache/zkapp-cli/keys/sandbox.json
-
-# Deploy zkApps
-echo "==================== DEPLOYING ZKAPPS ======================"
-echo "If this fails, it's likely due to incompatibility between the
-o1js and zkapp-cli versions in use."
-echo "NOTE: At the moment the daemon still has an old version of
-      snarkyjs pinned to it, so we cannot use the latest version of
-      zkapp-cli, which requires o1js. Once the pinned snarkyjs version
-      gets updated, this build will most likely fail and we will then
-      need to update the zkapp-cli version used here. This is
-      unfortunate, but necessary. Please delete this warning once it's
-      done." > /dev/stderr
-
-deploy_txs=()
-for zkapp_path in ${ZKAPP_PATH}/*/; do
-  zkapp_path=${zkapp_path%/}
-  zkapp=$(basename $zkapp_path)
-  echo "Deploying ${zkapp}..."
-
-  zkapp_account_pk=$(cat ${MINA_KEYS_PATH}/zkapp-${zkapp}-account.key.pub)
-  zkapp_account_privkey=$(mina-dev advanced dump-keypair --privkey-path "${MINA_KEYS_PATH}/zkapp-${zkapp}-account.key" | sed -ne "s/Private key: //p")
-
-  mkdir -p ${zkapp_path}/keys
-  echo -e "{\n    \"privateKey\": \"${zkapp_account_privkey}\",\n    \"publicKey\": \"${zkapp_account_pk}\"\n}" >"${zkapp_path}/keys/sandbox.json"
-
-  cat <<EOF >"${zkapp_path}/config.json"
-{
-  "version": 1,
-  "networks": {
-    "sandbox": {
-      "url": "http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql",
-      "keyPath": "keys/sandbox.json",
-      "feepayerKeyPath": "/root/.cache/zkapp-cli/keys/sandbox.json",
-      "feepayerAlias": "sandbox",
-      "fee": "1"
-    }
-  }
-}
-EOF
-  cd "$zkapp_path"
-  npm ci
-  npm run build
-  txn=$(zk deploy sandbox -y | sed -ne "s/https:\/\/berkeley.minaexplorer.com\/transaction\///p")
-  deploy_txs+=txn
-  cd -
-  echo "Done."
-done
+send_value_transfer_txns &
```

Comment:
Generating commands only makes sense if we run `rosetta-cli` `check:data` or `check:construction`.
```suggestion
if [[ "$MODE" == "full" ]]; then
  send_value_transfer_txns &
fi
```